spyder/plugins/variableexplorer.py

```diff
@@ -104,15 +104,41 @@ def __init__(self, parent):
     @staticmethod
     def get_settings():
         """
-        Return Variable Explorer settings dictionary
-        (i.e. namespace browser settings according to Spyder's configuration file)
+        Retrieve all Variable Explorer configuration settings.
+        
+        Specifically, return the settings in CONF_SECTION with keys in 
+        REMOTE_SETTINGS, and the setting 'dataframe_format'.
+        
+        Returns:
+            dict: settings
         """
+        section = VariableExplorer.CONF_SECTION
         settings = {}
-#        CONF.load_from_ini() # necessary only when called from another process
         for name in REMOTE_SETTINGS:
-            settings[name] = CONF.get(VariableExplorer.CONF_SECTION, name)
+            settings[name] = CONF.get(section, name)

```

Comment:
This is a static method so there is no `self`. I will convert it to a normal method but I discovered that it is called in a number of places so I need to change them ...